modulus/models/diffusion/preconditioning.py

```diff
@@ -584,11 +594,20 @@ def __init__(
         sigma_max=float("inf"),
         sigma_data=0.5,
         model_type="DhariwalUNet",
+        img_in_channels=None,
+        img_out_channels=None,
```

Comment:
I'm a little bit worried that this change might break some of the in-progress PRs for contributing other diffusion models to Modulus. I'll have to check